api/nodes/serializers.py

```diff
@@ -115,6 +116,11 @@ class NodeSerializer(JSONAPISerializer):
         related_meta={'count': 'get_registration_count'}
     ))
 
+    primary_institution = RelationshipField(
+        related_view='nodes:node-institution-detail',
+        related_view_kwargs={'node_id': '<pk>'}
+    )
```

Comment:
You have the functionality for adding a node institution but you need to give the user the link to do so.
```
primary_institution = RelationshipField(
        related_view='nodes:node-institution-detail',
        related_view_kwargs={'node_id': '<pk>'},
        self_view='nodes:node-relationships-institution',
        self_view_kwargs={'node_id': '<pk>'}
    )
```